testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -886,6 +886,8 @@
                                 <keycloak.authSession.map.storage.provider>jpa</keycloak.authSession.map.storage.provider>
                                 <keycloak.client.map.storage.provider>jpa</keycloak.client.map.storage.provider>
                                 <keycloak.clientScope.map.storage.provider>jpa</keycloak.clientScope.map.storage.provider>
+                                <keycloak.eventsStore.admin.map.storage.provider>jpa</keycloak.eventsStore.admin.map.storage.provider>
+                                <keycloak.eventsStore.auth.map.storage.provider>jpa</keycloak.eventsStore.auth.map.storage.provider>
```

Comment:
Could you please use a different separator than `.` for `eventsStore.admin`, e.g. `eventsStore-admin` or (perhaps better) even `adminEventsStore`?